### PR TITLE
docs/node16-apple-setup

### PR DESCRIPTION
**Pull request by KacperMadej**
Added docs for Node setup for Apple.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206408652981583

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -43,6 +43,12 @@ npm install
 gulp
 ```
 
+### Node setup for Apple Mx CPU
+
+Since [Node 16 TLS](https://nodejs.org/en/) is a ARM64 version available for Macs with Apple Ax CPU. Some node packages like [node-canvas](https://github.com/Automattic/node-canvas#compiling) with integrated compiling might fail. Install additional tools to resolve the problem:
```
Comment (goransle) [inline readme.md:48]: Perhaps we could simplify the first sentence to something like: 

> When running Node natively on ARM64 MacOS, some Node packages [...]

**Review by MarkusBarstad — APPROVED** [review]
I agree with Gørans comment, but good info!

**Review by goransle — APPROVED** [review]
LGTM

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**